settings.py

```diff
@@ -183,7 +183,28 @@
 # ex ADPOD_SLOTS = [1,2,3], will create 1st, 2nd and 3rd slot of adpod
 # ex ADPOD_SLOTS = [4,5], will create 4th and 5th slot of adpod
 # Slot numbers should be in incremental order.
-ADPOD_SLOTS = [1,2,3]
+ADPOD_SLOTS = [1]
```

Comment:
Why is this changed?

Reply:
This is just a sample config. I have set it to empty now , what kind of values are expected is mentioned in the comment above